expression/builtin_math.go

```diff
@@ -190,8 +192,85 @@ func builtinRound(args []types.Datum, ctx context.Context) (d types.Datum, err e
 
 // See http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_conv
 func builtinConv(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	//TODO  implement
-	return d, errors.New("Function unimplement")
+	var (
+		n        string
+		toBase   int64
+		fromBase int64
+		signed   bool
+		negative bool
+		touval   bool
+	)
+	for _, arg := range args {
+		if arg.IsNull() {
+			return d, nil
+		}
+	}
+	n, err = args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	sc := ctx.GetSessionVars().StmtCtx
+	fromBase, err = args[1].ToInt64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	toBase, err = args[2].ToInt64(sc)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	if fromBase < 0 {
+		fromBase = -fromBase
+		signed = true
+	}
+	if toBase < 0 {
+		touval = true
+		toBase = -toBase
+	}
+	if fromBase > 36 || fromBase < 2 || toBase > 36 || toBase < 2 {
+		return d, nil
+	}
+	n = getValidPrefix(n, fromBase)
+	if len(n) == 0 {
+		return d, nil
+	}
+	if n[0] == '-' {
+		negative = true
+		n = n[1:]
+	}
+
+	val, err := strconv.ParseUint(n, int(fromBase), 64)
+	if err != nil {
+		return d, errors.Trace(types.ErrOverflow)
+	}
+	//Ref https://github.com/mysql/mysql-server/blob/5.7/strings/ctype-simple.c#L598
```

Comment:
Add a space after the slash. And using `See` instead of `Ref` is better. 
The same as line258.